manifests/init.pp

```diff
@@ -361,7 +361,7 @@
   # system service configuration
   Boolean                        $service_enable                  = true,
   Enum['stopped', 'false', 'running', 'true'] $service_ensure     = 'running', # lint:ignore:quoted_booleans
-  Boolean                        $service_manage                  = true,
+  Boolean                        $service_manage                  = false,
```

Comment:
Is the change of the default value required? That would be a backwards incompatible change and require a major release.

Reply:
Yeah, it is. See comments in #283.

This may be one of those weird cases where it may not require a major version change though. Even though it appears puppet was managing this service, it never really has been since the release of 2.x.  It's been managed by the `gitlab-cl reconfigure` process that gets kicked by install or configuration changes. There would be any impact to implementors with this change.